Library/Formula/tig.rb

```diff
@@ -43,10 +43,18 @@ class Tig < Formula
   def install
     system "./autogen.sh" if build.head?
     system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{etc}"
-    system "make", "install"
+    system "make", "install", "sysconfdir=#{prefix/"share/tig/examples"}"
```

Comment:
Won't this make it rely on that directory for the config permanently, rather than just installing the sample?